localizations/zh-hans.json

```
@@ -0,0 +1,482 @@
+    "Preprocess images": "图像预处理",
+    "Name": "名称",
+    "Initialization text": "初始化文字",
+    "Number of vectors per token": "每个 token 的向量数",
+    "Overwrite Old Embedding": "覆写旧的 Embedding",
+    "Modules": "模块",
+    "Enter hypernetwork layer structure": "输入 hypernetwork 层结构",
+    "Select activation function of hypernetwork": "选择 hypernetwork 的激活函数",
+    "linear": "linear",
+    "relu": "relu",
+    "leakyrelu": "leakyrelu",
+    "elu": "elu",
+    "swish": "swish",
+    "Add layer normalization": "添加层标准化",
+    "Use dropout": "采用 dropout 防止过拟合",
+    "Overwrite Old Hypernetwork": "覆写旧的 Hypernetwork",
+    "Source directory": "源目录",
+    "Destination directory": "目标目录",
+    "Existing Caption txt Action": "对已有说明文字的 txt 的行为",
+    "ignore": "无视",
+    "copy": "复制",
+    "prepend": "放前面",
+    "append": "放后面",
+    "Create flipped copies": "生成镜像副本",
+    "Split oversized images into two": "将过大的图像分为两份",
+    "Split oversized images": "分割过大的图像",
+    "Use BLIP for caption": "使用 BLIP 生成说明文字(自然语言描述)",
+    "Use deepbooru for caption": "使用 deepbooru 生成说明文字(tags)",
+    "Split image threshold": "图像分割阈值",
+    "Split image overlap ratio": "分割图像重叠的比率",
+    "Preprocess": "预处理",
+    "Train an embedding; must specify a directory with a set of 1:1 ratio images": "训练 embedding； 必须指定一组具有 1:1 比例图像的目录",
+    "Train an embedding or Hypernetwork; you must specify a directory with a set of 1:1 ratio images": "训练 embedding 或者 hypernetwork； 必须指定一组具有 1:1 比例图像的目录",
+    "[wiki]": "[wiki]",
+    "Embedding": "Embedding",
+    "Embedding Learning rate": "Embedding 学习率",
+    "Hypernetwork Learning rate": "Hypernetwork 学习率",
+    "Learning rate": "学习率",
+    "Dataset directory": "数据集目录",
+    "Log directory": "日志目录",
+    "Prompt template file": "提示词模版文件",
+    "Max steps": "最大迭代步数",
+    "Save an image to log directory every N steps, 0 to disable": "每 N 步保存一个图像到日志目录，0 表示禁用",
+    "Save a copy of embedding to log directory every N steps, 0 to disable": "每 N 步将 embedding 的副本保存到日志目录，0 表示禁用",
+    "Save images with embedding in PNG chunks": "保存图像并在 PNG 文件中嵌入 embedding 文件",
+    "Read parameters (prompt, etc...) from txt2img tab when making previews": "进行预览时从文生图选项卡中读取参数（提示词等）",
+    "Train Hypernetwork": "训练 Hypernetwork",
+    "Train Embedding": "训练 Embedding",
+    "Create an aesthetic embedding out of any number of images": "从任意数量的图像中创建美术风格 embedding",
+    "Create images embedding": "生成图集 embedding",
+    "Apply settings": "保存设置",
+    "Saving images/grids": "保存图像/概览图",
+    "Always save all generated images": "始终保存所有生成的图像",
+    "File format for images": "图像的文件格式",
+    "Images filename pattern": "图像文件名格式",
+    "Always save all generated image grids": "始终保存所有生成的概览图",
+    "File format for grids": "概览图的文件格式",
+    "Add extended info (seed, prompt) to filename when saving grid": "保存概览时将扩展信息（随机种子、提示词）添加到文件名",
+    "Do not save grids consisting of one picture": "不要保存由一张图像组成的概览图",
+    "Prevent empty spots in grid (when set to autodetect)": "防止概览图中出现空白点（设置为自动检测时）",
```

```suggestion
    "Prevent empty spots in grid (when set to autodetect)": "（在自动检测时）防止概览图中出现空位",
```

虽然没找到哪里可以设置自动检测 \_(:з」∠)\_